docs/en/setup/service-agent/java-agent/Supported-list.md

```diff
@@ -29,6 +29,7 @@ metrics based on the tracing data.
   * [Apache httpcomponent AsyncClient](https://hc.apache.org/httpcomponents-asyncclient-4.1.x/) 4.x
   * [AsyncHttpClient](https://github.com/AsyncHttpClient/async-http-client) 2.1+
   * JRE HttpURLConnection (Optional²)
+  * [Hutool-http](https://www.hutool.cn/) 5.x
```

Comment:
According to [Hutool doc](https://www.hutool.cn/docs/#/http/Server/%E7%AE%80%E6%98%93Http%E6%9C%8D%E5%8A%A1%E5%99%A8-SimpleServer), it provides the capability as a hosting server, which we don't instrument in this PR.
So, should we polish this as

```suggestion
  * [Hutool-http](https://www.hutool.cn/) client 5.x
```